mautic/mautic
# Disable default values

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [y] 
| Related user documentation PR URL      | https://github.com/mautic/mautic-documentation/pull/321
| Related developer documentation PR URL | /
| Issue(s) addressed                     | /



#### Description:
This disables the default value for fields that Mautic uses to set someone as identified. This prevents the case where all visitors can be marked as identified if they have a default value set for first name. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Set a default value for first name
2. Visit a landing page or Mautic tracked page
3. Look for the contact in Mautic and notice they are not anonymous

#### Steps to test this PR:
1. Edit the custom field for first name and notice default value is disabled
2. Save and edit again and the default value should be empty
3. Edit email, company, first name, last name, any unique identifier field, or any social field and default value should be disabled. For company, companyname, companyemail, companywebsite, state, country and city should have default value disabled (all used to identify a company). 


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11250"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
